src/molecules/question/Question.tsx

```diff
@@ -259,14 +259,13 @@ export const QuestionData = ({
           control={control}
           rules={{ required: config?.mandatory }}
           render={({ field: { onChange, value } }) => {
-            const dateValue = value ? new Date(value) : null
-
             return (
-              <DatePicker
```

Comment:
[C] Can you remove the libraries we used for the old DatePicker from `package.json`?

Reply:
@nckhell I left it in as someone may be using the current implementation and we can't just deprecate it like that. What I can do is replace the internals of the DatePicker component we export to use the InputField without changing the props but any styling that someone has done over it will likely be impacted. WDYT?

Reply:
Done, removed the story altogether and added a new story under Input Field component, for dates

![image](https://user-images.githubusercontent.com/2485818/230403526-1a7f8f23-5525-4dcf-9d17-86f745100994.png)

See top comment screenshots, I've updated them